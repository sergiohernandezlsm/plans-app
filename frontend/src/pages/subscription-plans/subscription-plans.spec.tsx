import { shallow } from 'enzyme';
import SubscriptionPlansPage from './subscription-plans';

describe('SubscriptionPlansPage Component', () => {
  it('renders <SubscriptionPlansPage /> component', () => {
    const wrapper = shallow(<SubscriptionPlansPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
