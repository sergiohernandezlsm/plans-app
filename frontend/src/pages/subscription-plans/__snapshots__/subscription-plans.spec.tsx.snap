// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscriptionPlansPage Component renders <SubscriptionPlansPage /> component 1`] = `ShallowWrapper {}`;
